### ISSUE TITLE
[BUG]Bot fails to return to the previous question or parameter after pressing "Enter"

### ISSUE THREAD

**Issue by Clint-G13** [CoinAlpha/hummingbot#1606]
**Describe the bug**
When creating a pure market making strategy and asked about the bid or ask spread, press `enter` hoping to use the default value (1%) unfortunately bot won't enter any default value and it won't give any warning message but an error will appear on the log pane.

After pressing `enter`, bot won't return to the previous question and it will set all unfilled parameter to `None` value, and will resume only if you go back to the previous question using `config bid_spread` or `config ask_spread`

```
2020-04-03 05:45:32,784 - 2809 - hummingbot.client.hummingbot_application - ERROR - Unknown error while writing config. [<class 'decimal.ConversionSyntax'>]
Traceback (most recent call last):
  File "/home/clint_g/hummingbot/hummingbot/client/command/config_command.py", line 379, in _config_loop
    await self._inner_config_loop(keys)
  File "/home/clint_g/hummingbot/hummingbot/client/command/config_command.py", line 367, in _inner_config_loop
    await self._inner_config_loop(self._get_empty_configs())
  File "/home/clint_g/hummingbot/hummingbot/client/command/config_command.py", line 362, in _inner_config_loop
    value = await self.prompt_single_variable(cv, requirement_overwrite=False)
  File "/home/clint_g/hummingbot/hummingbot/client/command/config_command.py", line 318, in prompt_single_variable
    if not cvar.validate(val):
  File "/home/clint_g/hummingbot/hummingbot/client/config/config_var.py", line 46, in validate
    valid = self._validator(value)
  File "/home/clint_g/hummingbot/hummingbot/strategy/pure_market_making/pure_market_making_config_map.py", line 104, in <lambda>
    validator=lambda v: is_valid_decimal(v, 0, 99)),
  File "/home/clint_g/hummingbot/hummingbot/client/config/config_validators.py", line 32, in is_valid_decimal
    return Decimal(str(min_value)) <= Decimal(value) < Decimal(str(max_value))
decimal.InvalidOperation: [<class 'decimal.ConversionSyntax'>]
```
**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

Here are the steps to reproduce the issue (see attachments in the section below):
1. Create a pure market making strategy.
2. When asked to enter a bid or asked spread, press enter to use the default parameter (1%)
3. Check log pane and you will see an error.
4. Do `list configs` to verify parameters and you will see that all other parameters will be set to None.

**Screenshots**
The image below is taken after pressing Enter key
![Enter_default_value_cause_error](https://user-images.githubusercontent.com/57189990/78947922-5f261c80-7af9-11ea-9fab-582fee2a8dc7.png)

![a](https://user-images.githubusercontent.com/57189990/78948303-6863b900-7afa-11ea-8c30-94144c5f6a90.PNG)

**Release version**
```
Branch: Development
commit b421c3181589a27f6a938bd133f2d373dbad1a84
Date:   Wed Apr 1 10:45:37 2020 +0800
```
Note: Issue has been resolve on the future branch of Hummingbot.

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
**Optional: your ETH address:** (If you want to participate in the bug bounty)

**Comment by vic-en** [comment]
https://github.com/CoinAlpha/hummingbot/pull/1610

**Comment by dennisocana** [comment]
Confirmed fix in version 0.26.0
